src/main/java/com/conveyal/r5/analyst/scenario/Scenario.java

```diff
@@ -156,6 +156,16 @@ public TransportNetwork applyToTransportNetwork (TransportNetwork originalNetwor
         return copiedNetwork;
     }
 
+    /**
+    * @param lat Latitude of origin in floating point (not fixed int) degrees.
+    * @param lon Longitude of origin in floating point (not fixed int) degrees.
+    * @return time (in seconds) to wait at origin before beginning the trip
+    **/
+    public int computeOriginDelay (double lat, double lon) {
+        //TODO iterate through TNCPickupDelay modifications and return delay corresponding to lat/lon
```

Comment:
Is this method even used, or is it replaced by `getWaitTime()`?

Reply:
Deleted.